deploy/example/e2e_usage.md

```diff
@@ -1,14 +1,57 @@
 ## CSI driver E2E usage example
 ### Prerequisite
  - [Set up a Samba Server on a Kubernetes cluster](./smb-provisioner/)
-
-#### 1. Create PV/PVC bound with SMB share
- - Use `kubectl create secret` to create `smbcreds` with SMB username, password
+ > this example will create a new Samba Server(`//smb-server.default.svc.cluster.local/share`) with credential stored in secret `smbcreds`
+ - Use `kubectl create secret` to create `smbcreds` with Samba Server username, password
+ > skip this if already done
 ```console
 kubectl create secret generic smbcreds --from-literal username=USERNAME --from-literal password="PASSWORD"
 ```
 > add `--from-literal domain=DOMAIN-NAME` for domain support
 
+### Option#1: Storage Class Usage
+#### 1. Create a storage class
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: smb
+provisioner: smb.csi.k8s.io
+parameters:
+  source: "//smb-server.default.svc.cluster.local/share"
+  csi.storage.k8s.io/node-stage-secret-name: "smbcreds"
+  csi.storage.k8s.io/node-stage-secret-namespace: "default"
+  createSubDir: "true"  # optional: create a sub dir for new volume
```

Comment:
Is this actually supported yet? I don't see any implementation here: https://github.com/kubernetes-csi/csi-driver-smb/blob/master/pkg/smb/controllerserver.go#L29

Reply:
yes, there is a  trick here, I created a sub dir in NodeStageVolume:
https://github.com/kubernetes-csi/csi-driver-smb/blob/84ac39ff780b8db59b29f7f64d6dda2154356c13/pkg/smb/nodeserver.go#L95-L100

Reply:
Then there is no way to clean it up when the pvc gets deleted?

Reply:
yes, so `reclaimPolicy: Retain  # only retain is supported`